pkg/commands/workflows/plan_status_comments.go

```diff
@@ -84,7 +85,7 @@ func (c *PlanStatusCommentCommand) Flags() *cli.FlagSet {
 		Usage:   "The Guardian init job result status.",
 	})
 
-	f.StringVar(&cli.StringVar{
+	f.StringSliceVar(&cli.StringSliceVar{
```

Comment:
Why this change?

Reply:
tl;dr - we will have multiple jobs (plan and plan_destroy) to account for having to checkout different branches (pr sha for plan, main for plan_destroy). This means we will have an end status comment based on multiple job results.